src/pages/home/page.jsx

```diff
@@ -2,16 +2,10 @@ import React from "react";
 import { getData } from "../../common/request";
 
 
-let HomePage = React.createClass({
-  statics: {
-    fetchData(params) {
-      return getData("/home");
-    }
-  },
-
+export default class HomePage extends React.Component {
   componentWillMount() {
     console.log("[HomePage] will mount with server response: ", this.props.data.home);
```

Comment:
Yeah, the fat arrow syntax is currently not valid for class methods, and it seems like the current accepted solution[1] is to explicitly `bind()` the methods:

```js
class HomePage extends React.Component {
  constructor() {
    super();
    this.componentWillMount = this.componentWillMount.bind(this);
  }
```

Seems like with future property initializers you will be able to do:

```js
componentWillMount = () => {
```

The reason why I don't need to worry about it now for `componentWillMount` is because it is bound properly by `Component`. If I would be defining a custom method that I would want to call from `render()`, I'd need to bind it then.

[1] - http://facebook.github.io/react/blog/2015/01/27/react-v0.13.0-beta-1.html#autobinding